gremlin-console/src/main/static/NOTICE

```diff
@@ -18,15 +18,10 @@ Original source copyright:
 Copyright (c) 2008 Alexander Beider & Stephen P. Morse.
 
 ------------------------------------------------------------------------
-Apache Groovy 2.5.15 (AL ASF)
+Apache Groovy 4.0.9 (AL ASF)
 ------------------------------------------------------------------------
-This product includes/uses ANTLR (http://www.antlr2.org/)
-developed by Terence Parr 1989-2006
-
-This product bundles icons from the famfamfam.com silk icons set
-http://www.famfamfam.com/lab/icons/silk/
```

Comment:
Was this piece intended to be removed? Are the icons no longer bundled?

Reply:
The NOTICE file for groovy seems to have even more contents now: https://github.com/apache/groovy/blob/GROOVY_4_0_9/NOTICE

not sure what's actually packaged in the groovy jars though.

Reply:
> Was this piece intended to be removed? Are the icons no longer bundled?

The NOTICE file doesn't exist anymore in the groovy-console 4.0.9 jar, however, it does exist in https://github.com/apache/groovy/blob/GROOVY_4_0_9/subprojects/groovy-console/NOTICE so maybe we should add it back? In any case, groovy-console is only included with gremlin-server and not gremlin-console.

Reply:
> The NOTICE file for groovy seems to have even more contents now: https://github.com/apache/groovy/blob/GROOVY_4_0_9/NOTICE
> 
> not sure what's actually packaged in the groovy jars though.

I went through every groovy 4.0.9 jar included in our distribution and uber jars and they don't contain any more information than what I included in this PR. But, as I mentioned above, it is possible that they aren't packaging the NOTICE files in META-INF anymore so it might be better to just add everything from the NOTICE file you linked?

Reply:
my understanding is that for binary distributions, if the packaged jars do not have the NOTICE included then we do not need it in our NOTICE. if you've gone through all the groovy jars in both server/console and have only included what's there then i think we're good. 

the groovy NOTICE files we're looking at in code may be all source NOTICE-related which has a different set of rules to adhere to. we dont package groovy source so i think we're ok there.